### PR TITLE
Nightly

### PR DESCRIPTION
**Pull request by deviprasad97**
### Related Jira story
--------------------------

### Did you modify the folder `alectio_sdk`? If so, describe your changes in detail
-------------------------------------------------------------------------------------
Yes, Changed the backend IP, Changed Pipeline __init__ function added token argument, Added proper headers for OAuth authentication (Line 172). Updated all examples (Added the token=)

### If you have made any changes to an existing example, please go through the following list
--------------------------------------------------------------------------------------------------------
Updated all examples (Added the token=)

### Describe your change in detail and any other comments
------------------------------------------------------------
The token is attributed to my account.

### Describe any bugs or inconsistencies you may have encountered along the way
---------------------------------------------------------------------------------
N/A

### CODE REVIEW

File: alectio_sdk/flask_wrapper/pipeline.py [hunk 1]
```diff
@@ -58,11 +59,14 @@ def __init__(self, name, train_fn, test_fn, infer_fn, getstate_fn, args):
         self.infer_fn = infer_fn
         self.getstate_fn = getstate_fn
         self.args = args
-        self.client = S3Client()  # boto3.client('s3') #
-
-        dir_path = os.path.dirname(os.path.realpath(__file__))
-        with open(os.path.join(dir_path, "config.json"), "r") as f:
-            self.config = json.load(f)
```
Comment (arunsanknar) [inline alectio_sdk/flask_wrapper/pipeline.py:65]: self.config is still being used in pipeline.py. Did you comment this on purpose and was planning to use some other module to replicate the same?

File: setup.py [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+install_reqs = parse_requirements('./requirements.txt')
+
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+reqs = install_reqs
 setuptools.setup(
     name="alectio_sdk",
-    version="0.1.0",
+    version="0.0.3",
```
Comment (arunsanknar) [inline setup.py:16]: Make appropriate version releases , we are already at 0.1.0 , make an increment based on whether its a major or minor or a patch

**Comment by arunsanknar** [comment]
Some minor changes to requirements.txt under AlectioSDK/requirements
Add these two in the requirements.txt file
sentry-sdk==0.15.1
psutil==5.7.0

**Comment by deviprasad97** [comment]
Thank you. Fixed and updated it.

**Review by deviprasad97 — COMMENTED** [review]
Readme and version is now updated

**Review by arunsank — APPROVED** [review]

**Pull request merged**